package.json

```diff
@@ -1,108 +1,113 @@
-    "lint:css:all": "npm run lint:css -- ./**/*.scss",
-    "lint:js": "eslint --ext .js",
-    "lint:js:all": "npm run lint:js -- ./",
-    "lint": "npm-run-all lint:js:all lint:css:all",
-    "start": "npm-run-all -p start:dev watch:css watch:js",
-    "prestart": "npm run clean-build-dir",
-    "start:dev": "PORT=3000 NODE_ENV=development nodemon --inspect=0.0.0.0:9222 src --config nodemon.config.json",
-    "start:prod": "NODE_ENV=production node src",
-    "watch:css": "chokidar \"src/client/**/*.scss\" -c \"npm run build:css:dev\"",
-    "prewatch:css": "npm run build:css:dev",
-    "watch:js": "webpack"
-  },
-  "dependencies": {
-    "axios": "^0.19.2",
-    "cookie-session": "^1.3.3",
-    "core-js": "^3.4.7",
-    "dotenv": "^8.2.0",
-    "express": "^4.17.1",
-    "helmet": "^3.21.2",
-    "lodash": "^4.17.15",
-    "luxon": "^1.22.0",
-    "npm-run-all": "^4.1.5",
-    "prop-types": "^15.7.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-router-dom": "^5.1.2",
-    "regenerator-runtime": "^0.13.3",
-    "reselect": "^4.0.0",
-    "whatwg-fetch": "^3.0.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.7.4",
-    "@babel/plugin-proposal-class-properties": "^7.7.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.7.4",
-    "@babel/plugin-transform-runtime": "^7.7.4",
-    "@babel/preset-env": "^7.7.4",
-    "@babel/preset-react": "^7.7.4",
-    "@csstools/postcss-sass": "^4.0.0",
-    "assets-webpack-plugin": "^3.9.10",
-    "autoprefixer": "^9.7.3",
-    "babel-eslint": "^10.0.3",
-    "babel-jest": "^24.9.0",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-imports": "^2.0.0",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
-    "chokidar-cli": "^2.1.0",
-    "cssnano": "^4.1.10",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.15.1",
-    "enzyme-to-json": "^3.4.3",
-    "eslint": "^6.6.0",
-    "eslint-config-nava": "^8.0.0",
-    "eslint-config-prettier": "^6.6.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jest": "^23.0.4",
-    "eslint-plugin-jsdoc": "^18.0.1",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.16.0",
-    "eslint-plugin-react-hooks": "^2.3.0",
-    "husky": "^3.1.0",
-    "jest": "^24.9.0",
-    "jest-junit": "^9.0.0",
-    "lint-staged": "^9.4.3",
-    "nodemon": "^2.0.1",
-    "postcss-cli": "^6.1.3",
-    "postcss-hash": "^1.0.2",
-    "postcss-scss": "^2.0.0",
-    "prettier": "2.0.4",
-    "stylelint": "^12.0.0",
-    "stylelint-config-prettier": "^6.0.0",
-    "stylelint-junit-formatter": "^0.2.2",
-    "supertest": "^4.0.2",
-    "terser-webpack-plugin": "^2.2.1",
-    "webpack": "^4.41.2",
-    "webpack-bundle-analyzer": "^3.6.0",
-    "webpack-cli": "^3.3.10",
-    "webpackbar": "^4.0.0"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
+    "name": "cagov-benefits-screener",
```

Comment:
well, that's annoying. it's the linter though, so this is hopefully a one-off extra-indentation change?

Reply:
that's really weird... i wonder why this wasn't updated when i ran the linter the first time...

Reply:
If it changes back when you run the linter I will investigate :p

Reply:
If it changes back when you run the linter I will investigate :p